### ISSUE TITLE
Unable to alter SDDM settings from plasma settings

### ISSUE THREAD

**Issue by physkets** [artix-linux/world#47]
After a recent update, I suddenly saw that all my custom settings on SDDM were gone. I see that this is due to upstream wanting to use a new format or [something](https://github.com/sddm/sddm/issues/906). 

So then when I went to my desktop, and tried to set my theme and autologin preferences. But the settings wouldn't stick. Everytime I opened the settings dialog, the options would be back to default.

Is an issue in `sddm-kcm`? Is it supposed to create a `/etc/sddm` file for custom settings? It wasn't doing it in mine.

I then manually moved `/etc/sddm.conf.pacsave` to `/etc/sddm.conf`, and now my settings ae back.

**Comment by udeved** [comment]
sddm provide a default conf in /usr/lib/sddm/sddm.conf.d/sddm.conf which can be overridden by the user in /etc/sddm.conf.

`# Move sddm.conf to /usr/lib https://bugs.archlinux.org/task/56609`

**Comment by physkets** [comment]
Yes, true, but when I use plasma's settings dialog to make changes, it should create an `/etc/sddm.conf`, because that counts as user specified custom settings, and that is where custom settings should be. But this isn't happening (at least for me). This behaviour is, I think, controlled by the `sddm-kcm` package.
